client/packages/purchasing/src/purchase_order/DetailView/LineEdit/PurchaseOrderLineEditModal.tsx

```diff
@@ -60,7 +60,14 @@ export const PurchaseOrderLineEditModal = ({
       if (mode === ModalMode.Create) {
         await create();
       } else if (mode === ModalMode.Update) {
-        await update();
+        const res = await update();
+        const { success, error: updateError } = res;
+        if (!success) {
```

Comment:
Did this so that the modal doesn't close... wondering if they should be able to delete the line from this view as well since it shouldn't be in the order 🤔  would need designs though since this isn't a pattern we do with our current 'item edit' modals